packages/instant/src/components/erc20_asset_amount_input.tsx

```diff
@@ -8,13 +8,14 @@ import { BigNumberInput } from '../util/big_number_input';
 import { util } from '../util/util';
 
 import { ScalingAmountInput } from './scaling_amount_input';
-import { Container, Text } from './ui';
+import { Container, Flex, Icon, Text } from './ui';
 
 // Asset amounts only apply to ERC20 assets
 export interface ERC20AssetAmountInputProps {
     asset?: ERC20Asset;
     value?: BigNumberInput;
     onChange: (value?: BigNumberInput, asset?: ERC20Asset) => void;
+    onSymbolClick?: (asset?: ERC20Asset) => void;
```

Comment:
`onSymbolClick` feels like a bit of a generic name for me, as `Symbol` could mean many things.   What do you think about something like `onChangeAssetClick` or something that describes more what the user is intending to do?

Reply:
Yea I guess I was trying to keep it generic but by including the text "select token" in the component code that was a bit of a lie anyway.